src/renderer/shared/api/storage/service/dexie.ts

```diff
@@ -69,6 +69,8 @@ class DexieStorage extends Dexie {
       metadata: '++id',
     });
 
+    this.version(22).upgrade(removeMetadataDuplications);
```

Comment:
What about deleting all metadata?
It updates from time to time anyway, and we need to decide how to remove old ones in future